TAO/orbsvcs/ImplRepo_Service/ImR_Activator_i.cpp

```diff
@@ -139,8 +139,13 @@ ImR_Activator_i::init_with_orb (CORBA::ORB_ptr orb, const Activator_Options& opt
         "ImR_Activator");
       ACE_ASSERT (! CORBA::is_nil(this->imr_poa_.in ()));
 
+      obj = orb->resolve_initial_references ("POACurrent");
+      ACE_ASSERT (! CORBA::is_nil (obj.in ()));
```

Comment:
Is an assert the best way to handle this? The assert will abort the process and on some embedded platforms just make the system completely unresponsive